lib/suspenders/generators/app_generator.rb

```diff
@@ -253,5 +253,17 @@ def get_builder_class
     def using_active_record?
       !options[:skip_active_record]
     end
+
+    private
+
+    def honeybadger_outro
+      msg = "Run 'bundle exec honeybadger heroku install' with your API key"
+
+      if options[:heroku]
+        msg << " unless you're using the Heroku Honeybadger add-on"
+      end
+
+      "#{msg}."
```

Comment:
You could make the second part of the message a private method that returns the `unless you're using...` string or a blank string, and then `#honeybadger_outro` could look like this:

```ruby
    def honeybadger_outro
      "Run 'bundle exec honeybadger heroku install' with your API key#{honeybadger_message_suffix}."
    end

    def honeybadger_message_suffix
      if options[:heroku]
        " unless you're using the Heroku Honeybadger add-on"
      end
    end
```